src/views/ModelPlan/TaskList/Payment/ClaimsBasedPayment/index.tsx

```diff
@@ -0,0 +1,577 @@
+import GetClaimsBasedPayment from 'queries/Payments/GetClaimsBasedPayment';
+import {
+  GetClaimsBasedPayment as GetClaimsBasedPaymentType,
+  GetClaimsBasedPayment_modelPlan_payments as ClaimsBasedPaymentFormType,
+  GetClaimsBasedPaymentVariables
+} from 'queries/Payments/types/GetClaimsBasedPayment';
+import { UpdatePaymentsVariables } from 'queries/Payments/types/UpdatePayments';
+import UpdatePayments from 'queries/Payments/UpdatePayments';
+import { ClaimsBasedPayType, PayType } from 'types/graphql-global-types';
+import flattenErrors from 'utils/flattenErrors';
+import { sortOtherEnum, translateClaimsBasedPayType } from 'utils/modelPlan';
+import { NotFoundPartial } from 'views/NotFound';
+
+import { renderCurrentPage, renderTotalPages } from '..';
+
+const ClaimsBasedPayment = () => {
+  const { t } = useTranslation('payments');
+  const { t: h } = useTranslation('draftModelPlan');
+  const { modelID } = useParams<{ modelID: string }>();
+
+  const formikRef = useRef<FormikProps<ClaimsBasedPaymentFormType>>(null);
+  const history = useHistory();
+
+  const { data, loading, error } = useQuery<
+    GetClaimsBasedPaymentType,
+    GetClaimsBasedPaymentVariables
+  >(GetClaimsBasedPayment, {
+    variables: {
+      id: modelID
+    }
+  });
+
+  const {
+    id,
+    payType,
+    payClaims,
+    payClaimsOther,
+    payClaimsNote,
+    shouldAnyProvidersExcludedFFSSystems,
+    shouldAnyProviderExcludedFFSSystemsNote,
+    changesMedicarePhysicianFeeSchedule,
+    changesMedicarePhysicianFeeScheduleNote,
+    affectsMedicareSecondaryPayerClaims,
+    affectsMedicareSecondaryPayerClaimsHow,
+    affectsMedicareSecondaryPayerClaimsNote,
+    payModelDifferentiation
+  } = data?.modelPlan?.payments || ({} as ClaimsBasedPaymentFormType);
+
+  const modelName = data?.modelPlan?.modelName || '';
+
+  const [update] = useMutation<UpdatePaymentsVariables>(UpdatePayments);
+
+  const handleFormSubmit = (
+    formikValues: ClaimsBasedPaymentFormType,
+    redirect?: 'next' | 'back' | 'task-list'
+  ) => {
+    const { id: updateId, __typename, ...changeValues } = formikValues;
+    update({
+      variables: {
+        id: updateId,
+        changes: changeValues
+      }
+    })
+      .then(response => {
+        if (!response?.errors) {
+          if (redirect === 'next') {
+            history.push(
+              `/models/${modelID}/task-list/payment/anticipating-dependencies`
+            );
+          } else if (redirect === 'back') {
+            history.push(`/models/${modelID}/task-list/payment`);
+          } else if (redirect === 'task-list') {
+            history.push(`/models/${modelID}/task-list/`);
+          }
+        }
+      })
+      .catch(errors => {
+        formikRef?.current?.setErrors(errors);
+      });
+  };
+
+  const mappedClaimsBasedPayType = Object.keys(ClaimsBasedPayType)
+    .sort(sortOtherEnum)
+    .map(key => ({
+      value: key,
+      label: translateClaimsBasedPayType(key)
+    }));
```

Comment:
There is a util called `mapMultiSelectOptions` that does the work of this function for any Multiselect.  Implementation is as follows: 

```
options={mapMultiSelectOptions(
                      translateProviderAddType,
                      ProviderAddType
                    )}
```